app/views/declaration/additional_fiscal_references.scala.html

```diff
@@ -43,26 +43,57 @@
 
 @(mode: Mode, itemId:String, form: Form[AdditionalFiscalReference], additionalReferences: Seq[AdditionalFiscalReference] = Seq.empty)(implicit request: Request[_], messages: Messages)
 
+@referenceTable = {
+    @if(additionalReferences.nonEmpty){
+        <div class="field-group mb-2">
+            <table id="removable_elements">
+
+                <thead>
+                    <tr>
+                        <th id="@messages("declaration.additionalFiscalReferences.numbers.header")-heading" colspan="2">@messages("declaration.additionalFiscalReferences.numbers.header")</th>
+                    </tr>
+                </thead>
+
+                <tbody>
+                @for((elem, index) <- additionalReferences.map(_.asString).zipWithIndex) {
+                    <tr id="removable_elements__row@index">
+                        <td id="removable_elements-row@index-label">@{elem}</td>
+                        <td id="removable_elements-row@index-remove_button">
+                            <input
+                                type="submit"
+                                class="button--secondary"
+                                formmethod="post"
+                                formaction="@routes.AdditionalFiscalReferencesController.removeReference(mode, itemId, elem)"
+                                value="@messages("site.remove")"
+                            >
+                        </td>
+                    </tr>
+                }
+                </tbody>
+            </table>
+        </div>
+    }
+}
+
 @main_template(
     title = messages("declaration.additionalFiscalReferences.title")
 ) {
 
-    @components.back_link(routes.FiscalInformationController.displayPage(mode, itemId), mode)
+    @components.back_link(routes.FiscalInformationController.displayPage(mode, itemId))
```

Comment:
?

Reply:
Probably by accident - moreover in `back_link` template `mode` seems not used at all
https://github.com/hmrc/customs-declare-exports-frontend/blob/CEDS-1549/app/views/components/back_link.scala.html#L21

Reply:
Yep but it will be and this will likely cause a bug as we will forget it was taken off

Reply:
Fixed - in this case maybe not putting default one would be much better ?